net: lib: nrf_cloud: Clean up KConfigs so they don't clutter unrelated samples (#20074)
nrfconnect/sdk-nrf: doc/nrf/libraries/networking/nrf_cloud_alert.rst

Configuration
*************

> Enable :kconfig:option:`CONFIG_NRF_CLOUD` and configure the following Kconfig options to enable or disable the library and to select the data transport method:

Reviewer: ```suggestion
To enable this library, set the :kconfig:option:`CONFIG_NRF_CLOUD` Kconfig option and configure the following options to select the data transport method:
```
